### PR TITLE
Updated jwt gem and jwt options

### PR DESCRIPTION
**Pull request by grempe**
Second pull request. I closed the original pull request.  This should now merge cleanly.  I rebased on top of your master branch.

Thanks for the pull and a new release.

### CODE REVIEW

**Comment by grempe** [comment]
Any thoughts on this updated PR?

File: lib/rack/jwt/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Rack
-  module Jwt
+  module JWT
     VERSION = "0.1.1"
```
Comment (eigenbart) [inline lib/rack/jwt/version.rb]: version should be updated, renaming `Jwt` to `JWT` is not gonna be backwards-compatible, I think we should go for version 0.2.0

**Comment by grempe** [comment]
I added another commit to bump the version to 0.2.0 as requested.

**Comment by eigenbart** [comment]
thanks @grempe!!  I'll release a version of the gem

**Comment by grempe** [comment]
Great, thanks for pulling in the changes and releasing!

**Pull request merged**